squad/ci/models.py

```diff
@@ -20,11 +20,6 @@ def list_backends():
 class Backend(models.Model):
     name = models.CharField(max_length=128, unique=True)
     url = models.URLField()
-    # listener_url is used by backend's listen() method
-    listener_url = models.URLField(null=True, blank=True)
```

Comment:
I would prefer re-adding it later than keeping it there now without any function